src/modes/play/main.js

```diff
@@ -73,6 +73,10 @@ class PlayMode {
   playerNewLife() {
     this.cycleLevel();
   }
+
```

Comment:
I'm not exactly crazy about the idea that killing mobs === removing them from the mobs array, but it's the best plan I could think of, in that it covers all the bases. Open to other ideas.